db/migrate/20190928091301_create_page_sections.rb

```diff
@@ -1,16 +1,16 @@
 # Create table for page sections
 class CreatePageSections < ActiveRecord::Migration[6.0]
   def change
-    create_table :page_sections do |t|
+    create_table :page_sections, if_not_exists: true do |t|
       t.string :name, null: false
       t.text :description
       t.string :title, null: false
       t.string :slug, null: false
       t.integer :default_page_id
       t.integer :section_id
       t.integer :sort_order
+      t.boolean :hidden_from_menu, null: false, default: false
```

Comment:
Consider writing new migrations to modify existing tables.

Reply:
As discussed, the reason I keep editing old 'CreateTable' migrations instead of adding new 'AddColumn' ones is the lack of the AFTER keyword in Postgres, and my distaste for the disordered tables that result.

That said, I was thinking that dropping the whole table and recreating it with the new column would be a bad thing because data loss (!) but what if I renamed the current table, created the new one, and then copied the data across - are there Rails-y constructs for doing it that way?

Reply:
Here are the methods that would help with doing this: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html

Methods that look particularly relevant include `columns(table_name)`, `indexes(table_name)`, `rename_table`, `drop_table`, `rename_column`, `remove_column`.

Rather than recreating the table and indexes, it may be slightly less work doing the technique described here: https://wiki.postgresql.org/wiki/Alter_column_position#Add_columns_and_move_data